libraries/networking/src/udt/Socket.h

```diff
@@ -94,12 +100,12 @@ private slots:
     std::unordered_map<HifiSockAddr, SequenceNumber> _unreliableSequenceNumbers;
     std::unordered_map<HifiSockAddr, std::unique_ptr<Connection>> _connectionsHash;
     
-    QMutex _connectionsMutex; // guards concurrent access to connections hashs
-    
     int _synInterval = 10; // 10ms
-    QTimer _synTimer;
+    QTimer* _synTimer;
```

Comment:
It's owned by the QObject

On Tue, Sep 1, 2015 at 6:16 AM Clément Brisset <notifications@github.com>
wrote:

> In libraries/networking/src/udt/Socket.h
> <https://github.com/birarda/hifi/pull/40#discussion_r38411741>:
>
> >      int _synInterval = 10; // 10ms
> > -    QTimer _synTimer;
> > +    QTimer* _synTimer;
>
> We're leaking the timer here, we might want to use a unique_ptr
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/birarda/hifi/pull/40/files#r38411741>.
>